backend/controllers/deleteEmailDomain.js

```diff
@@ -24,13 +23,32 @@ const deleteEmailDomainFactory = (deleteEmailDomainApi, listEmailDomainsUrl) =>
     } catch (error) {
       if (error.status === 400) {
         res.redirect(req.originalUrl)
+      } else if (error.status === 401 && error.response && error.response.body) {
+        const domainUnauthorizedError = [
+          {
+            href: '#domainName',
+            text: 'Unauthorized to use the delete email domain functionality',
+          },
+        ]
+        stashStateAndRedirectToIndex(req, res, domainUnauthorizedError)
+      } else if (error.status === 403 && error.response && error.response.body) {
+        const domainRoleForbiddenError = [
+          {
+            href: '#domainName',
+            text: 'Forbidden to use the delete email domain functionality without the relevant role',
```

Comment:
Are these error conditions covered by authorisationMiddleware

Reply:
As discussed 403 is covered by the auth middleware but the other error scenarios will remain. Thanks Karen.